### ISSUE TITLE
Android TV support?

### ISSUE THREAD

**Issue by aicynide** [abdallahmehiz/mpvKt#52]
Is it fully supported in Android TV?

**Comment by Pentaphon** [comment]
I also want this to have full Android TV support.

**Comment by ArceyTheSecond** [comment]
> Is it fully supported in Android TV?

Yes, and no. It'll boot and work mostly, but on (for instance) the Nvidia Shield the gamepad seems only partly supported. Pausing the video is a little awkward, as there is no button mapped to do so. 

I too would love having (better) ATV support though! Mostly by for instance bringing up the menu when [D-pad DOWN], or pausing be pressing [A]. The app is also not recognised as an 'official' ATV capable app, so it's not addable to the regular home menu without 3rd party apps.

Loving it so far though! 💕

**Comment by MulverineX** [comment]
Even just rudimentary support for Android TV would be awesome for now; capture Back input to bring up the player controls, at least that way it would be usable.

Currently there isn't any perfect way to enjoy AV1 content on Jellyfin via Android TV devices. Kodi sync is slow, VLC has app switching jank, old MPV UI sucks, MX player has an old av1 decoder that lags sometimes, and Next Player doesn't support AV1 at all.

**Comment by Pentaphon** [comment]
> Currently there isn't any perfect way to enjoy AV1 content on Jellyfin via Android TV devices.

https://github.com/nova-video-player/aos-AVP works for that but I would like to try mpvKt with Nova on Android TV.
